### ISSUE TITLE
[São Paulo] Full Stack developer na Interledgerx

### ISSUE THREAD

**Issue by finhires** [frontendbr/vagas#792]
Tags: [Full Time] [JavaScript] [NodeJS] [Senior] [Pleno]

**Role:** Full Stack Developer position
**Location:** São Paulo, SP, Brazil (we are willing to help with housing and relocation costs for those coming from outside Sao Paulo)

**Industry:** Financial services

**Job Description:**
We seek strong developers to join us in our financial services technology startup. Our primary focus is on developing quality technology.

**Required skills:**
- Intellectual curiosity
- Comfortable with Linux/DevOps, vue.js, git, nodejs, test frameworks, CIs, expressjs, passportjs.
- Basic knowledge of cryptography, cybersecurity management
- Fluent in Javascript and at least one of the following languages (python, ruby, scala, go, elixir, erlang, haskell, rust, c++)
- Humility to ask for help when needed

**Desired:**
- Participated projects (former professional projects, personal projects, contributions to open source software, university groups, etc) and solid academic background are a plus, but not required.

**We offer a highly competitive compensation package:**
- Very competitive salary, commensurate with candidate's skill set
- End of year bonus package, commensurate with performance
    

Candidate will directly report to a highly experienced programmer, our hands-on CTO who loves to discuss programming language features and has used Linux daily for more than 12 years ;)

The bottom line is we are building solid technology in an environment that encourages openness, intellectual humility, and we are having a lot of fun doing it.

Please send an email with subject “[fullstack] <developer name>” to finhires@gmail.com, with attached CV and/or github profile link

**Comment by vinyguedess** [comment]
Aceita remoto ?

**Comment by eduardojmatos** [comment]
Essa vaga já está inativa por mais dias do que o permitido. Caso ainda tenha a vaga, reabra uma nova issue.

Mais informações: https://github.com/frontendbr/vagas#importante
